### ISSUE TITLE
feat(api/remove): expand `/remove/` endpoint to accept specific versions to remove

### ISSUE THREAD

**Issue by ramfox** [qri-io/qri#736]
need to allow for a `gen` param that specifies which version must be removed.
-1 means remove the whole dataset

**Comment by b5** [comment]
correct me if I'm wrong, but the command line uses `revisions` to specify the number of commits to delete?

**Comment by ramfox** [comment]
Yes, but I'm not sure how that relates to the frontend? Should we have a modal that slides out that says "how many versions do you want to delete"
Or just we just have a "undo" button that people can pop back
Or should we have a button next to each history that let's you revert back to that version as the head?

**Comment by ramfox** [comment]
Undo button would be the quickest an easiest, so that get's my vote for now.

- push the button => alert pops up "Are you sure you want to undo? This will remove the latest version of your dataset, your changes will be lost" => confirm => sends request to backend => on error, display the error, on success push to the previous version
